packages/block-editor/src/components/global-styles/hooks.js

```diff
@@ -42,6 +41,11 @@ const VALID_SETTINGS = [
 	'color.link',
 	'color.palette',
 	'color.text',
+	'color.gradient',
```

Comment:
Is this line intended? We already have `color.gradients` further up, so I wasn't sure if this was a typo.

Reply:
Yeah, I wasn't sure if it was a typo or a new setting given some of the odd requirements to disable color UIs. In terms of the [schema](https://github.com/WordPress/gutenberg/blob/77bb6c8f541d9c5097cf4865e478e04f69b20f7b/schemas/json/theme.json#L179) at least, it's not a valid setting. 

There are several other follow-on places that use `color.gradient` in this PR that would need tweaking if this is a typo.

The abundance of gradient settings might warrant clarification in our docs and the schema. We already have `gradients` and `customGradient` under settings. The block support flags use `gradients` as well, I believe, although there is a typo in the [block support docs](https://github.com/WordPress/gutenberg/blob/0c208a7a1f730b546ecb9f38bc7b22e1c892c3a4/docs/reference-guides/block-api/block-supports.md#colorgradients) that reference color.gradient.

Reply:
This is not a typo. It's been some time that I did this change but the idea is that `color.gradients` is the equivalent of `color.palette` for solids and that `color.gradient` is the equivalent of `color.text` or `color.background` (enable/disable these things)